src/main/resources/jcabigithub.properties

```diff
@@ -0,0 +1,4 @@
+#User-Agent info
+JCabi-Version ${project.version}
```

Comment:
@amihaiemil during the build, when exactly are these placeholders being replaced? I don't see it.

Reply:
@llorllale Those values are replaced automatically by ``Properties``, when ``load(...)`` is called. If they are not found, they will be empty string.

The properties are taken from pom.xml, but ``buildNumber`` is empty when unit testing, since it's only set by a maven plugin from ``jcabi-parent``, at build time:

```
<plugin>
    <!--
        This plugin will collect current build number
        from Git and save it into Maven property
   -->
   <groupId>org.codehaus.mojo</groupId>
   <artifactId>buildnumber-maven-plugin</artifactId>
   <version>1.3</version>
   <executions>
     <execution>
        <id>jcabi-build-number</id>
        <phase>initialize</phase>
        <goals>
            <goal>create</goal>
            <goal>create-timestamp</goal>
        </goals>
        <configuration>
            <buildNumberPropertyName>buildNumber</buildNumberPropertyName>
            ...
```

Reply:
@amihaiemil actually, I just realized it's `mvn` itself that filters the resources when copying over to the build directory. Anyways, it's fine.